packages/llama-index-auth0-ai/pyproject.toml

```diff
@@ -14,6 +14,7 @@ readme = "README.md"
 python = "^3.11"
 llama-index = "^0.12.11"
 openfga-sdk = "^0.9.0"
+auth0-ai = {path = "../auth0-ai", develop = true}
```

Comment:
is it ok to have develop when publishing?

Reply:
Yes and no. The develop dependency will stay, but should only be installed when working locally or running tests.

Additionally, we should add the published dependency.

This is done through groups, since I'm not sure what the publishing script is, I didn't want to change the groups, but for now I moved these dependencies to a dev group.

Before publishing, this has to be fixed